tests/fixtures/customer_cus_4QWKsZuuTHcs7X.json

```diff
@@ -1,7 +1,7 @@
 {
 	"id": "cus_4QWKsZuuTHcs7X",
 	"object": "customer",
-	"account_balance": 0,
+	"balance": 0,
```

Comment:
this is duplicating "balance" a few lines below.  Since this is just a snapshot of what Stripe's currently returning we should keep account_balance in the fixture for now (though we'll not use it).